modules/presets/index.js

```diff
@@ -60,6 +60,23 @@ function addHistoricalFields(fields) {
     fields.source.type = 'source';
     fields.source.source = false;
     fields.source.keys = ['source', 'source:url', 'source:name', 'source:date'];
+
+    const multipleSourceIds = ['', ':2', ':3', ':4'];
```

Comment:
Should we also support `source:1` when it’s present? [`source:1`](https://taginfo.openstreetmap.org/keys/source%3A1#overview) is even more common than [`source:3`](https://taginfo.openstreetmap.org/keys/source%3A3#overview). Or I guess we could deprecate it in favor of `source` and `source:2`, but then we’ll want to track that as a separate issue so we don’t end up skipping these sources.